modules/store-devtools/src/config.ts

```diff
@@ -104,6 +104,16 @@ export class StoreDevtoolsConfig {
    * Auto pauses when the extension’s window is not opened, and so has zero impact on your app when not in use.
    */
   autoPause?: boolean;
```

Comment:
According to redux doc, trace accept function.
https://github.com/reduxjs/redux-devtools/blob/main/extension/docs/API/Arguments.md#trace